addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -860,6 +860,28 @@ export function preserveCursor(document) {
     };
 }
 
+/**
+ * Check if the selection starts inside given selectors. This function can be
+ * used as the `isDisabled` property of a command of the PowerBox to disable
+ * a command in the given selectors.
+ * @returns {boolean} true selector is within one of the selectors
```

Comment:
Maybe add a @param to say that `selectors` should be one string with all desired selectors separated by a comma (multi-selector)